mpas_analysis/ocean/time_series_transport.py

```diff
@@ -0,0 +1,668 @@
+
+        fields = [transport]
+        lineColors = ['k']
+        lineWidths = [2.5]
+        legendText = [mainRunName]
+        if plotControl:
+            controlRunName = self.controlConfig.get('runs', 'mainRunName')
+            ref_transport, ref_mean, ref_std = \
+                self._load_transport(self.controlConfig)
+            fields.append(ref_transport)
+            lineColors.append('r')
+            lineWidths.append(1.2)
+            legendText.append(controlRunName)
+
+        if config.has_option(self.taskName, 'firstYearXTicks'):
+            firstYearXTicks = config.getint(self.taskName,
+                                            'firstYearXTicks')
+        else:
+            firstYearXTicks = None
+
+        if config.has_option(self.taskName, 'yearStrideXTicks'):
+            yearStrideXTicks = config.getint(self.taskName,
+                                             'yearStrideXTicks')
+        else:
+            yearStrideXTicks = None
+
+        fig = timeseries_analysis_plot(config, fields, movingAverageMonths,
+                                       title, xLabel, yLabel,
+                                       calendar=calendar,
+                                       lineColors=lineColors,
+                                       lineWidths=lineWidths,
+                                       legendText=legendText,
+                                       firstYearXTicks=firstYearXTicks,
+                                       yearStrideXTicks=yearStrideXTicks)
+
+        # do this before the inset because otherwise it moves the inset
+        # and cartopy doesn't play too well with tight_layout anyway
+        plt.tight_layout()
+
+        add_inset(fig, fc, width=2.0, height=2.0)
+
+        savefig(outFileName)
+
+        caption = 'Transport through the {} Transect'.format(title)
+        write_image_xml(
+            config=config,
+            filePrefix=filePrefix,
+            componentName='Ocean',
+            componentSubdirectory='ocean',
+            galleryGroup='Transport Time Series',
+            groupLink='transporttime',
+            thumbnailDescription=title,
+            imageDescription=caption,
+            imageCaption=caption)
+        # }}}
+
+    def _load_transport(self, config):  # {{{
+        """
+        Reads transport time series for this transect
+        """
+        # Authors
+        # -------
+        # Xylar Asay-Davis
+
+        baseDirectory = build_config_full_path(
+            config, 'output', 'timeSeriesSubdirectory')
+
+        startYear = config.getint('timeSeries', 'startYear')
+        endYear = config.getint('timeSeries', 'endYear')
+
+        inFileName = '{}/transport/transport_{:04d}-{:04d}.nc'.format(
+            baseDirectory, startYear, endYear)
+
+        dsIn = xarray.open_dataset(inFileName)
+        transport = dsIn.transport.isel(nTransects=self.transectIndex)
+        mean = transport.mean()
+        std = transport.std()
```

Comment:
@milenaveneziani, I'm computing the mean and standard deviation here but so far not adding them to the plot.  Could you show me where/how you wanted to display them?